spec/controllers/communities_controller_spec.rb

```diff
@@ -1,32 +1,8 @@
 require 'spec_helper'
 
 describe CommunitiesController, :type => :controller do
-  routes { Rails.application.class.routes }
```

Comment:
Yeah....Have no idea whats going on here? We are doing a ton of setup only to do a smoke test on the controller. Let's remove this useless setup.